enterprise-suite/scripts/lbc.py

```diff
@@ -410,29 +544,47 @@ def setup_args(argv):
                         action='store_true')
 
     # Install arguments
-    install.add_argument('--dry-run', help='only print out the commands that will be executed',
-                        action='store_true')
     install.add_argument('--force-install', help='set to true to delete an existing install first, instead of upgrading',
                         action='store_true')
     install.add_argument('--export-yaml', help='export resource yaml to stdout',
                         choices=['creds', 'console'])
-    install.add_argument('--helm-name', help='helm release name', default='enterprise-suite')
+    install.add_argument('--reuse-resources', help='try to reuse PVCs and cluster roles from a previous install',
```

Comment:
May not be realistic, but is it conceivable someone will only want to reuse only one of PVCs or roles?  How would that work?

Reply:
That should work - when this flag is set, it checks for existing PVCs and cluster roles separately. If they don't exist, it still creates them. So it could, for example, reuse PVCs and create cluster roles. If no reusable resources exist, flag does nothing besides checking.

Reply:
Made the help message more clear about this.